src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 4.2.0
 -----
 
+ * [BC BREAK] `SenderLocator` has been renamed to `ContainerSenderLocator`
+   Be careful as there is still a `SenderLocator` class, but it does not rely on a ContainerInterface to find senders
```

Comment:
```patch
- but it does not rely on a ContainerInterface to find senders
+ but it does not rely on a `ContainerInterface` to find senders. Instead, it accepts the sender instance itself instead of its identifier in the container.
```

?

Reply:
done